DNN Platform/Modules/Groups/GroupsModuleBase.cs

```diff
@@ -268,9 +268,9 @@ public bool GroupListUserGroupsOnly
         #endregion
 
         #region Public Methods
-        public string GetCreateUrl()
+        public string GetCreateUrl(string closingUrl)
         {
-            return ModuleContext.EditUrl("Create"); //.NavigateUrl(GroupCreateTabId,"",true,null);
+            return ModuleContext.EditUrl(string.Empty, string.Empty, "Create", closingUrl, Array.Empty<string>());
```

Comment:
Since `closingUrl` will be ignored if popups are turned off, this doesn't fix the issue all of the time (cf. https://github.com/dnnsoftware/Dnn.Platform/blob/8a88f9af173213d9386920d2f7ee87ec8e06a208/DNN%20Platform/Library/UI/Modules/ModuleInstanceContext.cs#L833)

It would probably be better to issue a `Response.Redirect` after creating a group, rather then relying on the popup to take you somewhere upon closing.

Reply:
I agree, it would solve it when not using popPups too.

Reply:
ok. let me test the scenario. Thanks.

Reply:
@bdukes , 

Thanks your brilliant recommendation. I have tested with the Response.Redirect and it works for both. I will close this PR and create a new PR to fix with Response.Redirect. Thanks.